Gemfile

```diff
@@ -10,6 +10,14 @@ source 'http://repo.release.cerner.corp/internal/rubygems/' if internal
 gem 'berkshelf', '~> 3.0', group: :deployment
 gem 'rubocop', '~> 0.18'
 gem 'foodcritic', '~> 4.0'
+gem 'rspec',  '~> 3.1'
+
+# https://github.com/opscode/chef/issues/2547
+if Bundler.current_ruby.on_19?
```

Comment:
Is this part of the context of this change?

I am concerned on this and would like to better understand:
* What version of ruby currently comes with ChefDK?
* Will this force users local version of Chef onto Chef 12 when they want to stay on Chef 11 in anyway?